routing/payment_lifecycle.go

```diff
@@ -535,48 +516,6 @@ func (p *paymentLifecycle) requestRoute(ps *paymentState) (*route.Route,
 	return nil, nil
 }
 
-// stop signals any active shard goroutine to exit and waits for them to exit.
-func (p *paymentLifecycle) stop() {
-	close(p.quit)
-	p.wg.Wait()
-}
-
-// waitForShard blocks until any of the outstanding shards return.
-func (p *paymentLifecycle) waitForShard() error {
```

Comment:
Could be beneficial for review to remove in the exact commit where it becomes unused, to see the bigger picture better?